### PR TITLE
created admin profile section

### PR DESCRIPTION
**Pull request by sourabh-sawant-v2**
1. Added: Profile section for the admin is added.
2. Added:  Two custom is methods are added which allows admin to change password, with some steps taken ensure admin is 
                 authenticated before user able change password. 
3. Added: Two new forms are added in view, to allow admin to update profile and change password.
4. Added: Routes are added to access custom methods in controller.
5. Removed: Index method for admin is removed, to maintain data security of admin credentials of other admins, which were 
                     accessible prior.

### CODE REVIEW

File: app/controllers/admins_controller.rb [hunk 1]
```diff
@@ -20,12 +21,15 @@ def new
   def edit
   end
 
+  def edit_custom
```
Comment (shilesh) [inline app/controllers/admins_controller.rb]: Why this action is called `edit_custom` . Should be `edit` ?

File: app/views/admins/edit_custom.html.erb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<h1>Change Account Password</h1>
```
Comment (shilesh) [inline app/views/admins/edit_custom.html.erb]: `edit_password ` can be directly used with out edit custom ?

File: config/routes.rb [hunk 3]
```diff
@@ -8,7 +8,13 @@
   resources :addresses
   resources :locations
 
-  resources :admins  
+  resources :admins do 
+    member do
+      get 'edit_custom'
+      patch 'change_password', to: 'admins#change_password', as: :changepass
```
Comment (shilesh) [inline config/routes.rb]:
```suggestion
      patch 'change_password', to: 'admins#change_password'
```
change_password is meaningful than `changepass`

**Review by shilesh — CHANGES_REQUESTED** [review]
comments

File: app/controllers/admins_controller.rb (hunk 1 above)
Comment (sourabh-sawant-v2) [reply app/controllers/admins_controller.rb]: addition action edit_custom is removed.

File: app/views/admins/edit_custom.html.erb (hunk 2 above)
Comment (sourabh-sawant-v2) [reply app/views/admins/edit_custom.html.erb]: edit_password is used with existing edit action.

File: config/routes.rb (hunk 3 above)
Comment (sourabh-sawant-v2) [reply config/routes.rb]: changepass changed to change_password

**Pull request merged**